app/views/job_actions/_form.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class="col-md-8">
+<%= form_for @job_action, html: { id: 'job_action_form' } do |f| %>
+  <%= text_f f, :name %>
+  <%= selectable_f f, :job_template_id, JobTemplate.all.map { |t| [t.name, t.id] },
```

Comment:
permission check needed when loading `JobTemplate.all`

Reply:
also this may be a long list, perhaps grouping per job category would be better, but only if easily possible with select2